rest_framework_json_api/relations.py

```diff
@@ -274,7 +273,7 @@ def get_resource_type_from_included_serializer(self):
                 inflection.singularize(field_name),
                 inflection.pluralize(field_name),
             ]
-            includes = get_included_serializers(parent)
+            includes = getattr(parent, "included_serializers", dict())
```

Comment:
I copied this from another inline comment which got outdated by a commit but actually was not resolved:

From an API perspective this is a bit ugly that `getattr` always needs to be called instead of simply `serializer.included_serializers`. I would prefer that `included_serializers` is set to `None` when not configured so it easier to read `included_serializers` for a DJA user.

Easiest I guess is to set `included_serializers` in the serializer base class to `None`.

Reply:
I looked into this some more as well. Internally we will need to leave `getattr` anyway because DJA allows mixed serializers and DRF serializers won't have `included_serializers` set.

I'd also played around with the API to allow defaults. Quite a few changes would be needed, more than I have anticipated for this to work. I think this crosses the boundaries of this PR so let's just leave it as you had it for now. We certainly need to think about the DJA serializer API more broadly but in a different thread.